src/lib.rs

```diff
@@ -59,16 +59,24 @@
 //!   starvation
 //!
 //! - `std` enables support for thread yielding instead of spinning
+//!
+//! - `portable-atomic` enables usage of the `portable-atomic` crate
+//!   to support platforms without native atomic operations (Cortex-M0, etc.).
+//!   The `portable_atomic_unsafe_assume_single_core` or `critical-section` feature
```

Comment:
I think I should make this kind of update in multiple places:

```suggestion
//!   Either one of the `unsafe-assume-single-core` or `critical-section` features (not both)
```

Reply:
That seems sensible!